refocus-bdk-server.js

```diff
@@ -485,19 +485,42 @@ module.exports = (config) => {
     }, HEARTBEAT_TIMER);
   } // heartBeat
 
+  /**
+   * Gets room type information
+   * @param {*} roomTypeId;
+   * @returns {object} Roomtype information or null
+   */
+  async function getRoomTypeById(roomTypeId) {
+    try {
+      if (roomTypeId) {
+        // eslint-disable-next-line max-len
+        const response = await generic.get(`${SERVER}${API}${ROOM_TYPES_ROUTE}/${roomTypeId}`,
+          TOKEN, DEFAULT_TRIES, null, PROXY_URL);
+        return response.body;
+      }
+    } catch (error) {
+      return null;
+    }
+    return null;
+  }
+
   /**
    * Checks RoomType bot list for bot triggering event
    * @param {Object} event - refocusEvent
    * @param {String} bot - botName
    * @returns {boolean}
    */
-  function isBotInstalledInRoom(event, bot) {
-    const botsInRoom = event.Room.RoomType.Bots;
-    const botsInstalled = botsInRoom.find((b) => b.name === bot);
-    if (botsInstalled) {
-      return true;
+  async function isBotInstalledInRoom(event, bot) {
+    const eventRoomTypeId = event.Room.RoomType.id;
+    try {
+      const botsInRoomType = await getRoomTypeById(eventRoomTypeId);
+      if (botsInRoomType) {
```

Comment:
if here is false, does the method returns false without an explicit return?

Reply:
Actually, it will return an undefined, as we're not explicitly returning anything, but It should return false, tough. Missed a return false to set the default response as false. I think we can test this case by stubbing the getRoomTypeById() with a null response. I'll try to cover this case and validate my assumption. thanks :)

Reply:
added the return false and a test for a scenario where botsInRoomType is null